modules/lib/build.gradle

```diff
@@ -139,6 +175,8 @@ subprojects {
     task prepareToPublish( type: Copy, dependsOn: processResources ) {
         def fullName = project.name
         def shortName = project.name.substring( 4 )
+        // TODO: Replace with the actual shared dependencies in the future, like ['@enonic-types/core']
```

Comment:
there should be any todo left

Reply:
It's just a reminder for [this](https://github.com/enonic/xp/issues/9729) task, so there be no miss understanding, why the following array is actually empty. The actual list of the libs that will use shared types is not final yet.